src/generated/apiFetcher.ts

```diff
@@ -136,7 +136,7 @@ type JobResult = {
   };
 };
 
-async function loadJob(signal: AbortSignal | undefined, delayedJobId: string): Promise<JobResult> {
+async function loadJob(signal: AbortSignal | undefined, delayedJobId: string, retries: number = 3): Promise<JobResult> {
```

Comment:
🎨 There's no reason to specify the type when a default value is provided; TS compiler will do just fine understanding it.

Reply:
removed